src/olympia/reviewers/tests/test_views.py

```diff
@@ -1981,7 +1981,17 @@ def test_only_viewable_with_specific_permission(self):
     def test_results(self):
         self.login_with_permission()
         self.generate_files()
-        self._test_results()
+        with self.assertNumQueries(0):
```

Comment:
`0`?

Reply:
That was for testing, it should be 23 as well...